lib/rajska.ex

```diff
@@ -61,11 +68,8 @@ defmodule Rajska do
 
   defmacro __using__(opts \\ []) do
     all_role = Keyword.get(opts, :all_role, :all)
-    roles = Keyword.get(opts, :roles)
-    roles_with_tier = add_tier_to_roles!(roles)
-    roles_names = get_role_names(roles)
-    super_roles = get_super_roles(roles_with_tier)
-
+    valid_roles = Keyword.get(opts, :valid_roles, [])
+    super_roles = Keyword.get(opts, :super_roles, [])
```

Comment:
Before we had `super_roles` as a list because we could have roles with the same tier. Does it still make sense now that tiers were removed?

Or maybe we should keep `super_roles` but still have tiers? I'm not sure

Reply:
I think a `super_role` atom makes more sense